src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -59,6 +60,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
                 subdirApp.UseEndpoints(endpoints =>
                 {
+                    endpoints.Map("NotAComponent.html", context => context.Response.WriteAsync("<div id=\"test-info\">Not a component!</div>"));
```

Comment:
Is this line still needed, consider that you added `src/Components/test/testassets/BasicTestApp/wwwroot/NotAComponent.html`?